### DISCUSSION TITLE
How can I automatically trigger a page break?

### DISCUSSION THREAD

**Discussion by kmink3225** [quarto-dev/quarto-cli#6284] [Q&A] [answered]
### Description

Hello, 

first of all, thank you for providing dynamic documentation open source like quarto. I have one question on using it.

When rendering a long table or multiple tables, the result is cut off.  
How can I automatically trigger a page break?

For reproducible code,

````
---
format: pdf
engine: knitr
---

```{r}
library(knitr)
```

```{r}
#| tbl-cap: "iris"
#| tbl-subcap:
#| - "iris1"
#| - "iris2"
#| - "iris3"
#| - "iris4"

kable(iris[1:20,])
kable(iris[21:40,])
kable(iris[41:60,])
kable(iris[61:80,])
```
````

Output

![image](https://github.com/quarto-dev/quarto-cli/assets/61118160/c24e8e78-db4e-4180-96ed-ad67e52795fd)

**Comment by mcanouil** [comment, marked as answer]
See <https://quarto.org/docs/authoring/markdown-basics.html#page-breaks>.


Note that since multiple sub-tables are supposed to be together by definition, you might not be able to add a page break.
In my opinion you should not use sub-tables for long tables because if you have your fourth long table on the fifth page, you no longer get the full table caption making pointless the sub-table design.

**Reply by kmink3225** [reply]
Thank you for your answer. I'll make sure to keep in mind your answers!
